rviz_plugins/eStop_plugin/manifest.xml

```diff
@@ -0,0 +1,19 @@
+<package>
```

Comment:
This is fine for now but we should change it so there is only one manifest file and one plugin description file for all of the plugins like this https://github.com/ros-visualization/visualization_tutorials/tree/groovy-devel/rviz_plugin_tutorials